### PR TITLE
[mkldnn-1.0] add skipped case for mkldnn_v1.0

### PR DESCRIPTION
**Pull request by rongzha1**
## Description ##
add skipped case for mkldnn_v1.0

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at https://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here
@TaoLv @juliusshufan @PatricZhao @ElaineBao @ciyongch

### CODE REVIEW

**Comment by wuxun-zhang** [comment]
Maybe we also need enable unittests for quantized op here (link:https://github.com/apache/incubator-mxnet/blob/mkldnn-v1.0/tests/python/mkl/test_quantization_mkldnn.py#L27).

**Comment by rongzha1** [comment]
> Maybe we also need enable unittests for quantized op here (link:https://github.com/apache/incubator-mxnet/blob/mkldnn-v1.0/tests/python/mkl/test_quantization_mkldnn.py#L27).

Exactly right! Thanks for your kind reminder.

**Comment by pengzhao-intel** [comment]
@rongzha1 I have merged the dependent PRs and please rebase this PR and run all test cases again.

File: tests/python/quantization/test_quantization.py [hunk 1]
```diff
@@ -747,6 +747,7 @@ def get_fp32_sym_with_multiple_outputs(length=1):
     return sym
 
 @with_seed()
+@unittest.skip('skip for MKL-DNN 1.0 integration: https://github.com/apache/incubator-mxnet/projects/16')
```
Comment (pengzhao-intel) [inline tests/python/quantization/test_quantization.py]: @ZhennanQin could we switch quantization tests now? I see several INT8 PRs are already merged.

Comment (rongzha1) [reply tests/python/quantization/test_quantization.py]: rebase and enable now, disable due to subgrap fc PR dependence

**Comment by pengzhao-intel** [comment]
@rongzha1 please take a look CI again, seems the community resolved part of CI issues.

**Comment by pengzhao-intel** [comment]
Merging now to avoid the flaky test.

**Pull request merged**